### PR TITLE
Add Admixer to amp-ad

### PR DESCRIPTION
**Pull request by AdmixerTech**

### CODE REVIEW

File: ads/admixer.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/admixer.js]: nit: 2018

File: ads/admixer.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  const imp = {
+    params: {
+      zone: data.zone,
+    },
+  };
+  if (data.sizes) {
+    imp.sizes = tryParseJson(data.sizes);
+  }
+  payload.imps.push(imp);
+  writeScript(global, 'https://inv-nets.admixer.net/ampsrc.js?data=' + JSON.stringify(/** @type {JsonObject} */ (payload)));
```
Comment (zhouyx) [inline ads/admixer.js]: nit: 80 chars limit

File: ads/admixer.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/admixer.md]: nit: 2018

File: ads/admixer.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+limitations under the License.
+-->
+
+# Admixer
+
+## Example
+
+```html
+<amp-ad width="300" height="250"
+          type="admixer"
```
Comment (zhouyx) [inline ads/admixer.md]: nit: indentation.

File: examples/ads.amp.html [hunk 5]
```diff
@@ -324,6 +325,17 @@ <h2>AdmanMedia</h2>
       data-id="8e916419">
   </amp-ad>
 
+  <h2>Admixer</h2>
+  <amp-ad width="300" height="250"
+          type="admixer"
```
Comment (zhouyx) [inline examples/ads.amp.html]: nit: indentation.

File: examples/ads.amp.html [hunk 6]
```diff
@@ -324,6 +325,17 @@ <h2>AdmanMedia</h2>
...
+          data-sizes="[[300,250]]"
+          data-zone="2eb6bd58-865c-47ce-af7f-a918108c3fd2">
+  </amp-ad>
+  <amp-ad type="admixer" width="300" height="300" layout="responsive"
+          data-sizes="[[300,250],[300,300]]"
```
Comment (zhouyx) [inline examples/ads.amp.html]: nit: indentation

File: ads/_config.js [hunk 7]
```diff
@@ -127,6 +127,14 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  admixer: {
+    renderStartImplemented: true,
```
Comment (zhouyx) [inline ads/_config.js]: When I test with your ad examples, I couldn't get the render-start message. Could you confirm that renderStartAPI is implemented? Same with noContentAvailable API.

**Comment by AdmixerTech** [comment]
@zhouyx `renderStart` API and `noContentAvailable` API is implemented

**Comment by zhouyx** [comment]
@AdmixerTech Looks great! I've restarted the percy test. Let's see if it fix the test failure. If not, please rebase to latest master to fix the test. Thanks.

**Comment by zhouyx** [comment]
@AdmixerTech Thanks for the Pull Request! PR merged.

**Pull request merged**